src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactVersionVulnerabilityScoreTest.java

```diff
@@ -0,0 +1,127 @@
+            VULNERABILITIES.value(new Vulnerabilities(vulnerability)),
+            VERSION.value("2.0.0")));
+  }
+
+  @Test
+  public void testThreeLowSeverityVulnerability() {
+    Vulnerability vulnerabilityOne =
+        TestUtils.createBasicVulnerability(2.0, "1.0.0", "2.2.0");
+    Vulnerability vulnerabilityTwo =
+        TestUtils.createBasicVulnerability(2.0, "1.0.0", "2.2.0");
+    Vulnerability vulnerabilityThree =
+        TestUtils.createBasicVulnerability(2.0, "1.0.0", "2.2.0");
+    assertScore(
+        4.0,
+        new ArtifactVersionVulnerabilityScore(),
+        Utils.setOf(
+            VULNERABILITIES.value(
+                new Vulnerabilities(vulnerabilityOne, vulnerabilityTwo, vulnerabilityThree)),
+            VERSION.value("2.0.0")));
+  }
+
+  @Test
+  public void testMidSeverityVulnerability() {
+    Vulnerability vulnerabilityOne =
+        TestUtils.createBasicVulnerability(5.0, "1.0.0", "2.2.0");
+    assertScore(
+        5.0,
+        new ArtifactVersionVulnerabilityScore(),
+        Utils.setOf(
+            VULNERABILITIES.value(
+                new Vulnerabilities(vulnerabilityOne)),
+            VERSION.value("2.0.0")));
+  }
+
+  @Test
+  public void testThreeMidSeverityVulnerability() {
+    Vulnerability vulnerabilityOne =
+        TestUtils.createBasicVulnerability(4.5, "1.0.0", "2.2.0");
+    Vulnerability vulnerabilityTwo =
+        TestUtils.createBasicVulnerability(5.0, "1.0.0", "2.2.0");
+    Vulnerability vulnerabilityThree =
+        TestUtils.createBasicVulnerability(6.0, "1.0.0", "2.2.0");
+    assertScore(
+        Score.MIN,
+        new ArtifactVersionVulnerabilityScore(),
+        Utils.setOf(
+            VULNERABILITIES.value(
+                new Vulnerabilities(vulnerabilityOne, vulnerabilityTwo, vulnerabilityThree)),
+            VERSION.value("2.0.0")));
+  }
+
+  @Test
+  public void testHighSeverityVulnerability() {
+    Vulnerability vulnerabilityOne =
+        TestUtils.createBasicVulnerability(9.0, "1.0.0", "2.2.0");
+    assertScore(
+        Score.MIN,
+        new ArtifactVersionVulnerabilityScore(),
+        Utils.setOf(
+            VULNERABILITIES.value(
+                new Vulnerabilities(vulnerabilityOne)),
+            VERSION.value("2.0.0")));
+  }
+
+  @Test
+  public void testMixedSeverityVulnerability() {
+    Vulnerability vulnerabilityOne =
+        TestUtils.createBasicVulnerability(2.0, "1.0.0", "2.2.0");
+    Vulnerability vulnerabilityTwo =
+        TestUtils.createBasicVulnerability(6.0, "1.0.0", "2.2.0");
+    Vulnerability vulnerabilityThree =
+        TestUtils.createBasicVulnerability(9.0, "1.0.0", "2.2.0");
+    assertScore(
+        Score.MIN,
+        new ArtifactVersionVulnerabilityScore(),
+        Utils.setOf(
+            VULNERABILITIES.value(
+                new Vulnerabilities(vulnerabilityOne, vulnerabilityTwo, vulnerabilityThree)),
+            VERSION.value("2.0.0")));
+  }
+
+  @Test
+  public void testNoVulnerabilityEtAll() {
```

Comment:
Not sure whether it is a typo or not but I like it.

Reply:
Was intended and nice that you like it 🙂
However I still prefer to not prefix test methods with `test` 😉